sdk/cmake/baseaddress64.cmake

```diff
@@ -0,0 +1,397 @@
+set(baseaddress_untfs                          0x7fe5c000000)
+set(baseaddress_updspapi                       0x7fe5b000000)
+set(baseaddress_url                            0x7fe5a000000)
+set(baseaddress_usrmgr                         0x7fe59000000)
+set(baseaddress_vbscript                       0x7fe58000000)
+set(baseaddress_vdmdbg                         0x7fe57000000)
+set(baseaddress_verifier                       0x7fe56000000)
+set(baseaddress_vssapi                         0x7fe55000000)
+set(baseaddress_w32time                        0x7fe54000000)
+set(baseaddress_wbemdisp                       0x7fe53000000)
+set(baseaddress_wbemprox                       0x7fe52000000)
+set(baseaddress_wdmaud.drv                     0x7fe51000000)
+set(baseaddress_win32u                         0x7fe50000000)
+set(baseaddress_win32u_2k3sp2                  0x7fe4f000000)
+set(baseaddress_win32u_xpsp2                   0x7fe4e000000)
+set(baseaddress_windowscodecs                  0x7fe4d000000)
+set(baseaddress_windowscodecsext               0x7fe4c000000)
+set(baseaddress_wined3dcfg                     0x7fe4b000000)
+set(baseaddress_winemp3.acm                    0x7fe4a000000)
+set(baseaddress_winfax                         0x7fe49000000)
+set(baseaddress_wing32                         0x7fe48000000)
+set(baseaddress_winhttp                        0x7fe47000000)
+set(baseaddress_winprint                       0x7fe46000000)
+set(baseaddress_winscard                       0x7fe45000000)
+set(baseaddress_winsta                         0x7fe44000000)
+set(baseaddress_wkssvc                         0x7fe43000000)
+set(baseaddress_wlanapi                        0x7fe42000000)
+set(baseaddress_wldap32                        0x7fe41000000)
+set(baseaddress_wlnotify                       0x7fe40000000)
+set(baseaddress_wmi                            0x7fe3f000000)
+set(baseaddress_wmisvc                         0x7fe3e000000)
+set(baseaddress_wmiutils                       0x7fe3d000000)
+set(baseaddress_wmvcore                        0x7fe3c000000)
+set(baseaddress_wshirda                        0x7fe3b000000)
+set(baseaddress_wshom                          0x7fe3a000000)
+set(baseaddress_wshtcpip                       0x7fe39000000)
+set(baseaddress_wsock32                        0x7fe38000000)
+set(baseaddress_wtsapi32                       0x7fe37000000)
+set(baseaddress_wuapi                          0x7fe36000000)
+set(baseaddress_wuauserv                       0x7fe35000000)
+set(baseaddress_xinput1_1                      0x7fe34000000)
+set(baseaddress_xinput1_2                      0x7fe33000000)
+set(baseaddress_xinput1_3                      0x7fe32000000)
+set(baseaddress_xinput9_1_0                    0x7fe31000000)
+set(baseaddress_xmllite                        0x7fe30000000)
+set(baseaddress_xolehlp                        0x7fe2f000000)
+set(baseaddress_zipfldr                        0x7fe2e000000)
```

Comment:
What about the 3 missing modules:
set(baseaddress_fusion
set(baseaddress_fusion_1_1
set(baseaddress_fusion_2_0
?

Reply:
The missing modules weren't in the past neither. So they doesn't matter for now.

Reply:
yes but they generate this extra spam.

Reply:
That has absolutely zero to do with this fix. Just because a PR touches file x, it doesn't mean that is has to fix every little shit related to file x.

Reply:
> The missing modules weren't in the past neither.

They are present in the old file. Possibly manually added?

Reply:
> That has absolutely zero to do with this fix

This has a lot to do with this PR, because you removed them. Just look at the file before your PR!

Reply:
There were less missing base-addresses than before. There have been missing base-addresses for x86 as well. And fusion was missing in all the other files. There were also base-addresses for test dlls, which shouldn't have been there. 
But fine, I fixed all of that as well. This makes the whole PR a lot bigger.

Reply:
You addressed my concern regarding the fusion part now. I do consider this discussion-thread as being resolved. Last question in that context: When reusing the 1 base address for all 3 fusion dlls... Are we sure, that they will never be loaded at the same time? Aim is avoiding relocations. Thx for your answer in advance.